app/services/bookings/gitis/fake_crm.rb

```diff
@@ -1,123 +1,16 @@
 module Bookings::Gitis
-  module FakeCrm
-    KNOWN_UUID = "b8dd28e3-7bed-4cc2-9602-f6ee725344d2".freeze
-    REQUIRED = %w{
-      firstname lastname emailaddress2 telephone2 birthdate
-      address1_line1 address1_city address1_stateorprovince address1_postalcode
-      dfe_channelcreation dfe_hasdbscertificate
-      dfe_Country@odata.bind
-    }.freeze
-    ALLOWED = (
-      REQUIRED + %w{
-        telephone1 address1_telephone1 address1_line2 address1_line3
-        emailaddress1 dfe_dateofissueofdbscertificate
-        dfe_PreferredTeachingSubject01@odata.bind
-        dfe_PreferredTeachingSubject02@odata.bind
-      }
-    ).freeze
-
-    def find_by_email(address)
-      return super unless stubbed?
-      return nil if address =~ /unknown/
-
-      Contact.new(fake_contact_data).tap do |contact|
-        contact.email = address
-      end
+  class FakeCrm < CRM
+    def initialize
+      @store = Store::Fake.new
     end
 
     def find_contact_for_signin(email:, firstname:, lastname:, date_of_birth:)
-      return super unless stubbed?
+      return super unless store.is_a?(Store::Fake)
```

Comment:
should this not raise if trying to use FakeCrm with a real store?

Reply:
Yeh good catch - this was a hangover from when I start splitting stuff apart